js/editor-ta.js

```diff
@@ -6,30 +6,27 @@
 function EditorTextArea(editorElement, settings) {
   this.element_ = editorElement;
   this.settings_ = settings;
-  this.numLines_ = 1;
-  this.currSession_ = null;
+  this.numLines_ = 1
   this.textareaPadding_ = 20;
   this.dimentions = {
     height: null,
     width: null
   };
+  this.setTheme();
   const initFontSize = this.settings_.get('fontsize') + 'px';
 
-  // The main text area the user will be interacting with.
-  // TODO: shift this into the HTML and reference it.
-  this.textarea_ = document.createElement('textarea');
-  this.textarea_.setAttribute('id', 'editor-textarea');
-  this.textarea_.style.fontSize = initFontSize;
-  this.textarea_.addEventListener('input', () => {
+  // We need a named reference to this arrow function so we can remove it
```

Comment:
this comment confused me, for some reason i thought it was saying we want to set it as an event handler, then we can remove it from some other thing it's attached to. perhaps reword to 'so we can remove it as an EventListener'.

Reply:
Will change :)